components/cards/EventCard.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <nuxt-link
+    :to="to"
+    class="card-link"
+  >
+    <article class="event-card">
+      <div class="event-card__content">
+        <header>
+          <span class="event-card__subtitle">
+            {{ type }}
+          </span>
+          <h3 class="event-card__title">
+            {{ title }}
+          </h3>
+        </header>
+
+        <footer>
+          <div class="event-card__info-detail">
+            <Map20 class="event-card__icon-map" />
+            <span class="event-card__place">{{ place }}</span>
+          </div>
+          <div class="event-card__date-and-arrow">
+            <div class="event-card__info-detail">
+              <Calendar20 class="event-card__icon-calendar" />
+              <span class="event-card__date">{{ date }}</span>
+            </div>
+            <ArrowRight20 class="event-card__icon-arrow-right" />
+          </div>
+        </footer>
+      </div>
+
+      <div
+        class="event-card__media"
+        :style="`background-image: ${decorate(image)};`"
+      />
+    </article>
+  </nuxt-link>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Prop } from 'vue-property-decorator'
+
+@Component
+export default class extends Vue {
+  @Prop(String) type
+  @Prop(String) title
+  @Prop(String) image
+  @Prop(String) place
+  @Prop(String) date
+  @Prop(String) to
+
+  decorate (image) {
+    const bgEffects = [
+      `url(${image})`
+    ]
+    return bgEffects.join(', ')
+  }
+}
+</script>
+
+<style lang="scss" scoped>
```

Comment:
Using BEM properly as you are doing, there is no need to apply `scoped` to the styles.

Reply:
Let's keep this. I cannot assure we are using BEM everywhere. We still need to decide something regarding #326